website/package.json

```diff
@@ -3,22 +3,37 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "docusaurus-start",
-    "build": "docusaurus-build",
-    "gh-pages": "docusaurus-publish",
-    "examples": "docusaurus-examples",
-    "crowdin-upload": "crowdin --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus-write-translations",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version"
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
+    "crowdin:upload": "cd .. && crowdin upload sources --config crowdin-v2.yaml",
+    "crowdin:downloadTranslations": "cd .. && crowdin download --config crowdin-v2.yaml",
+    "write-translations": "docusaurus write-translations",
+    "fetchSupporters": "node fetchSupporters.js",
+    "netlify:ci:production": "yarn netlify:prepare && yarn netlify:crowdin && yarn build",
+    "netlify:ci:deployPreview": "yarn netlify:prepare && yarn build -l en",
+    "netlify:prepare": "yarn fetchSupporters && yarn build:js",
+    "netlify:crowdin": "yarn write-translations && cd .. && yarn crowdin:upload && yarn crowdin:download"
   },
   "dependencies": {
-    "docusaurus": "^1.14.2"
+    "@crowdin/cli": "^3.5.2",
```

Comment:
no idea if dev vs not matters, but some of the ones added here seems more dev than not?

Reply:
moved

probably does not matter much because we build a static app, and the dependencies to build on netlify and for a dev to work are quite similar anyway. Separating the 2 is useful if you plan to do "yarn install --production", but in this case we need Crowdin in Netlify anyway, so it does not matter much and we can't speed the CI install process